src/renderers/testing/stack/ReactTestRendererStack.js

```diff
@@ -50,17 +50,21 @@ function getRenderedHostOrTextFromComponent(component) {
   return component;
 }
 
+var UNSET = {};
+
 class ReactTestComponent {
   _currentElement: ReactElement;
   _renderedChildren: null | Object;
   _topLevelWrapper: null | ReactInstance;
   _hostContainerInfo: null | Object;
+  _nodeMock: Object;
 
   constructor(element: ReactElement) {
     this._currentElement = element;
     this._renderedChildren = null;
     this._topLevelWrapper = null;
     this._hostContainerInfo = null;
+    this._nodeMock = UNSET;
```

Comment:
welp, I’ve updated it to on `mountComponent` because that is the earliest point we can do so (that is when `hostContainerInfo` is passed in)